pkg/core/native/oracle.go

```diff
@@ -308,10 +308,10 @@ func (o *Oracle) finish(ic *interop.Context, _ []stackitem.Item) stackitem.Item
 // FinishInternal processes an oracle response.
 func (o *Oracle) FinishInternal(ic *interop.Context) error {
 	if len(ic.VM.Istack()) != 2 {
-		return errors.New("Oracle.finish called from non-entry script")
+		return errors.New("Oracle.Finish called from non-entry script")
```

Comment:
It's important to keep it as is, this is a publicly (as in manifest) known method name.